babel/src/main/codegen/config.fmpp

```diff
@@ -112,6 +114,7 @@ data: {
       "SEMI"
       "SUBSTR"
       "TEMP"
+      "_TO_"
```

Comment:
Why do we need the "_TO_" keyword here (and elsewhere)? It doesn't look like the parser is explicitly looking for it during parsing.

Reply:
Oh sorry, this needs to be clean up and I was going to a meeting so I submitted this PR without looking at it too detailed. Will fix!

Reply:
Fixes in https://github.com/googleinterns/calcite/pull/84/commits/811eb1403bd7cb2f2d8a2a43fd72f2e1054e1090.